sys-fs/cryfs/cryfs-9999.ebuild

```diff
@@ -40,6 +40,9 @@ src_prepare() {
 	# remove tests that require internet access to comply with Gentoo policy
 	sed -i -e '/CurlHttpClientTest.cpp/d' -e '/FakeHttpClientTest.cpp/d' test/cpp-utils/CMakeLists.txt || die
 
+	# remove non-applicable warning caused by cmake-utils setting NDEBUG
+	sed -i -e '/WARNING! This is a debug build. Performance might be slow./d' src/cryfs-cli/Cli.cpp || die
```

Comment:
I assume upstream is counting on the user following their build instructions (or using their precompiled Ubuntu binaries), in which case this warning doesn't appear because CMAKE_BUILD_TYPE=Release results in `-O3 -NDEBUG` being used. I haven't submitted a bug or patch but I'm assuming it'd be blamed on the Gentoo build replacing their flags.